spec/lib/audit/checksum_spec.rb

```diff
@@ -1,14 +1,48 @@
 require 'rails_helper'
 require_relative "../../../lib/audit/checksum.rb"
+require_relative '../../load_fixtures_helper.rb'
+
 # FIXME: remove this rubocop once we start writing tests
 # TODO: implement this;  we begin with a placeholder
```

Comment:
I imagine we can remove the above comments?  Look for any others marked TODO or FIXME in the spec and in the class?